src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout-state/index.ts

```diff
@@ -12,7 +12,7 @@ export default Shopware.Component.wrapComponentConfig({
             type: String as PropType<'loading' | 'error' | 'success'>,
             required: true,
         },
-        errorSnippet: {
+        error: {
             type: String,
             required: false,
             default: 'errorSubtitle',
```

Comment:
so you're really defaulting in L75
```suggestion
            default: null,
```